database.c

```diff
@@ -180,14 +180,18 @@ static int do_sleep(struct timeval *delta) {
 #endif
 }
 
-static void print_replay_statistics() {
+static void print_replay_statistics(int dry_run) {
 	int hours, minutes;
 	double seconds, runtime, session_time, busy_time;
 	struct timeval delta;
 	unsigned long histtotal =
 		stat_hist[0] + stat_hist[1] + stat_hist[2] + stat_hist[3] + stat_hist[4];
 
+	if (0 == dry_run) {
 	fprintf(sf, "\nReplay statistics\n");
+	} else {
+		fprintf(sf, "\nReplay statistics (dry run)\n");
+	}
```

Comment:
That would be better like this:

    fprintf(sf, "\nReplay statistics%s\n", (dry_run ? " (dry run)" : ""));

But I'd just omit it.

Reply:
ack